src/index.js

```diff
@@ -1,13 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter as Router } from 'react-router-dom';
+import reportWebVitals from './reportWebVitals';
```

Comment:
- [ ] [Optional] remove this line as it is part of react boiler plate generated with npx create react-app, also clean up your index.js file and remove commented codes